test/distributed/test_device_mesh.py

```diff
@@ -462,7 +462,9 @@ def test_device_mesh_parent_child_hash(self):
         ep_mesh_2 = DeviceMesh(self.device_type, mesh_group_2)
         ep_mesh = ep_mesh_1 if self.rank < self.world_size // 2 else ep_mesh_2
         # ep_mesh is considered different from mesh_2d["TP"]
-        self.assertEqual(mesh_2d["TP"]._flatten_mesh_list, ep_mesh._flatten_mesh_list)
+        self.assertEqual(
```

Comment:
This is expected because these two meshes are not from same root mesh, so we just compare its mesh tensor directly.